### DISCUSSION TITLE
Alignment with USDOT WZDx

### DISCUSSION THREAD

**Discussion by schnuerle** [openmobilityfoundation/curb-data-specification#53] [Policy]
**See Issue #74 now for future discussion.**

How could CDS align with the USDOT's Work Zone Data Exchange (WZDx)? 

This came up in a discussion of lane types around a curb, vehicles that could be double parking or blocking curbs, and where and how to specify or align with WZDx IDs.

https://github.com/openmobilityfoundation/curb-data-specification/discussions/37#discussioncomment-1671646

**Comment by bhamlinSDOT** [comment]
I recently had some discussions with SDOT staff that are involved with the WZDx and there seems to be a number of areas where CDS could align with the WZDx.

* WZDx is currently using GeoJSON polyline and multi-point features rather than polygons mostly due to the fact that most agencies don’t have polygon data for their roads yet. WZDx doesn’t exclude polygon data since it’s just using a GeoJSON feature.
* Look at how WZDx road events that would close curbspaces and understand how an agency can build a streamlined workflow to communicate these closures in both feeds.
* Explore is scheduling, which by nature is quite built out in CDS, but where WZDx is still struggling to define how it will communicate these; particularly is cases where one part of a complex schedule changes… how is that communicated?
* WZDx has a lot of working ongoing surrounding communication of worker present and other ‘live’ data directly from field devices. It looks like CDS has thought this through with the Event object, much of which could translate to WZDx.

**Comment by schnuerle** [comment]
See Issue #74 for future discussions on how we align this to CDS in a future release.

This discussion thread is locked in favor of #74.
